tests/rt.conf

```diff
@@ -39,39 +39,39 @@ RUN | cpld_restart_c192_p8                              | - wcoss2 jet acorn gae
 RUN | cpld_bmark_p8                                     | - s4 jet cheyenne acorn    | baseline |
 RUN | cpld_restart_bmark_p8                             | - s4 jet cheyenne acorn    |          | cpld_bmark_p8
 
-COMPILE | 3 | intel | -DAPP=S2SW -DCCPP_SUITES=FV3_GFS_v17_coupled_p8 | | fv3 |
+COMPILE | p8_s2sw | intel | -DAPP=S2SW -DCCPP_SUITES=FV3_GFS_v17_coupled_p8 | | fv3 |
 RUN | cpld_control_noaero_p8                            |                            | baseline |
 RUN | cpld_control_nowave_noaero_p8                     |                            | baseline |
 
-COMPILE | 4 | intel | -DAPP=S2SWA -DDEBUG=ON -DCCPP_SUITES=FV3_GFS_v17_coupled_p8,FV3_GFS_cpld_rasmgshocnsstnoahmp_ugwp | - wcoss2 acorn | fv3 |
+COMPILE | p8_s2swa_debug | intel | -DAPP=S2SWA -DDEBUG=ON -DCCPP_SUITES=FV3_GFS_v17_coupled_p8,FV3_GFS_cpld_rasmgshocnsstnoahmp_ugwp | - wcoss2 acorn | fv3 |
 RUN | cpld_debug_p8                                     | - wcoss2 acorn             | baseline |
 
-COMPILE | 5 | intel | -DAPP=S2SW -DDEBUG=ON -DCCPP_SUITES=FV3_GFS_v17_coupled_p8 | - wcoss2 acorn | fv3 |
+COMPILE | p8_noaero_debug | intel | -DAPP=S2SW -DDEBUG=ON -DCCPP_SUITES=FV3_GFS_v17_coupled_p8 | - wcoss2 acorn | fv3 |
 RUN | cpld_debug_noaero_p8                              | - wcoss2 acorn             | baseline |
 
 # Waves and aerosol off for computing fluxes in mediator
-COMPILE | 6 | intel | -DAPP=S2S -DCCPP_SUITES=FV3_GFS_v17_coupled_p8_sfcocn -DCMEPS_AOFLUX=ON | | fv3 |
+COMPILE | p8_s2s_cmeps | intel | -DAPP=S2S -DCCPP_SUITES=FV3_GFS_v17_coupled_p8_sfcocn -DCMEPS_AOFLUX=ON | | fv3 |
 RUN | cpld_control_noaero_p8_agrid                      |                            | baseline |
 
-COMPILE | 7 | intel | -DAPP=S2S -DCCPP_SUITES=FV3_GFS_v17_coupled_p8 | | fv3 |
+COMPILE | p8_s2s | intel | -DAPP=S2S -DCCPP_SUITES=FV3_GFS_v17_coupled_p8 | | fv3 |
 RUN | cpld_control_c48                                  |                            | baseline |
 RUN | cpld_warmstart_c48                                |                            | baseline |
 RUN | cpld_restart_c48                                  |                            |          | cpld_warmstart_c48
 
-COMPILE | 8 | intel | -DAPP=S2SWA -DCCPP_SUITES=FV3_GFS_v17_coupled_p8,FV3_GFS_cpld_rasmgshocnsstnoahmp_ugwp -DFASTER=ON | - cheyenne | fv3 |
+COMPILE | p8_faster | intel | -DAPP=S2SWA -DCCPP_SUITES=FV3_GFS_v17_coupled_p8,FV3_GFS_cpld_rasmgshocnsstnoahmp_ugwp -DFASTER=ON | - cheyenne | fv3 |
 RUN | cpld_control_p8_faster                            | - cheyenne                 | baseline |
 
 # Unstructured WW3 mesh
-COMPILE | 9 | intel | -DAPP=S2SW -DCCPP_SUITES=FV3_GFS_v17_coupled_p8 -DPDLIB=ON | + hera orion cheyenne | fv3 |
+COMPILE | p8_pdlib | intel | -DAPP=S2SW -DCCPP_SUITES=FV3_GFS_v17_coupled_p8 -DPDLIB=ON | + hera orion cheyenne | fv3 |
```

Comment:
I would suggest ``p8_s2sw_pdlib`` and similar for the debug.

Reply:
Done.